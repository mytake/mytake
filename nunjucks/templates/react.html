{% extends "base.html" %}

{% block endbody %}
  <!-- Dependencies -->
  <script src="https://unpkg.com/react@15/dist/react.js"></script>
  <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>

  <!-- Main -->
  <script src="./bundle-0.0.3.js"></script>
{% endblock %}
