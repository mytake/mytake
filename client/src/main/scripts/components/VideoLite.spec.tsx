import * as React from "react";
import * as renderer from "react-test-renderer";
import VideoLite from "./VideoLite";
import { videoFactFast } from "../utils/testUtils";

jest.mock("react-youtube", () => ({
  default: "YouTube"
}));

test("Video", () => {
  const tree = renderer
    .create(
      <VideoLite
        videoId={videoFactFast.youtubeId}
        clipRange={[10, 20]}
        isFixed={false}
        onScroll={jest.fn()}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
