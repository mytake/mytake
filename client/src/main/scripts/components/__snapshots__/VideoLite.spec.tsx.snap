// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video 1`] = `
<YouTube
  className="video__video"
  onPause={[Function]}
  onReady={[Function]}
  onStateChange={[Function]}
  opts={
    Object {
      "height": "315",
      "playerVars": Object {
        "autoplay": 1,
        "cc_lang_pref": "en",
        "cc_load_policy": 1,
        "controls": 1,
        "end": 20,
        "modestbranding": 1,
        "playsinline": 1,
        "rel": 0,
        "showinfo": 0,
        "start": 10,
      },
      "width": "560",
    }
  }
  videoId="ApTLB76Nmdg"
/>
`;
